draft-regext-rdap-x-media-type.md

```diff
@@ -189,11 +189,36 @@ Provisional Registration: No
 
 # Acknowledgements
 
-Pawel Kowalik provided ideas and feedbacks that have contributed to
+Pawel Kowalik and James Mitchel have provided ideas and feedbacks that have contributed to
 the content of this document.
 
 {backmatter}
 
+# Using the Vary Header {#vary_header}
+
+Server implementers may want to consider using the `vary` header depending on the caching
+behavior desired of shared caches (i.e. middleboxes, not client caches).
+
+Consider the following scenario where user Bob and user Alice send queries to the same
+RDAP server that is routed through a middlebox network element implementing a shared HTTP caceh.
+
+User Bob sends a query for the domain `example.com`
+(http://regy.example/domain/example.com) without RDAP-x. The `accept` header sent
+for Bob's query would be `accept: application/rdap+json` or `accept: application/json`.  
+User Alice sends a query for the same domain, however her client uses RDAP-X. The `accept`
+header returned by Alice might be `accept: application/rdap-x+json, application/rdap+json`.
+
+If no `vary` header is set in the response for these queries, the shared cache will compare only
+the URL of the query when processing cache items and therefore user Bob and user Alice would receive
+the same answer. In other words, since both queried `http://regy.example/domain/example.com` the shared
+cache would return the answer of the first query to the second query and all other subsequent queries
+until the item expired out of the cache.
+
+If server implementers do not desire this behaviour and would rather caches consider each query
```

Comment:
How about "would rather signal that caches"?

Reply:
done